app/helpers/statistics_helper.rb

```diff
@@ -0,0 +1,15 @@
+module StatisticsHelper
```

Comment:
Rename and move to `lib/statistics.rb` please
```suggestion
module Statistics
```